src/Signup/FormComponent.jsx

```diff
@@ -39,10 +29,13 @@ const FormComponent = () => {
           <label>Password</label>
           <input type="text" name="password" />
         </div>
-        <button className="submitButton" type="submit">Signup</button>
+        <div className="buttonGroup">
+          <button type="submit" className="submitButton">Join</button>
+          <button type="button" className="cancelButton" onClick={() => window.close()}>Cancel</button>
```

Comment:
The more I think about it, the more I'm thinking we should remove this Cancel button. It doesn't add anything to the interview, and it's just another thing for the interviewee to get tripped up on

Reply:
You got a point!!   removing this!